CHANGELOG.md

```diff
@@ -6,19 +6,27 @@ This release prepares the core protocol for veRSR through the introduction of 3
 
 The release also expands collateral decimal support from 18 to 21, with some caveats about minimum token value. See [docs/solidity-style.md](./docs/solidity-style.md#Collateral-decimals) for more details.
 
+Finally, it adds resistance to toxic issuance by charging more when the collateral is under peg. This has the side-effect of also creating a new revenue source.
+
 ## Upgrade Steps
 
 TODO
 
 Make sure distributor table sums to >10000.
 
+Upgrade ALL collateral plugins
+
 ## Core Protocol Contracts
 
 - `AssetRegistry`
   - Prevent registering assets that are not in the `AssetPluginRegistry`
   - Add `validateCurrentAssets() view`
 - `BackingManager`
   - Switch from sizing trades using the low price to the high price
+- `BasketHandler`
+  - Increase `quote()` quantities during issuance to include an issuance premium for de-pegged tokens
+  - Add `setSkipIssunancePremium()`, callable by governance
```

Comment:
The setter is a double negative (Also a typo), ideally use something like `setIssuancePremiumEnabled` instead.

Reply:
That's fair -- I want the variable to keep the signage it has but I'll change the setter and event